my-app-router/src/services/getPosts.jsx

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import { Posts_URL } from '../constants/server';
+
```

Comment:
Make one universal functions, where you get argument url and make request on api with that url